ffmpeg/nodes.py

```diff
@@ -51,8 +52,21 @@ class FilterNode(Node):
     """FilterNode"""
     def _get_filter(self):
         params_text = self._name
-        arg_params = ['{}'.format(arg) for arg in self._args]
-        kwarg_params = ['{}={}'.format(k, self._kwargs[k]) for k in sorted(self._kwargs)]
+
+        # Helper function to escape uncomfortable characters
+        def escape_chars(seq, keys):
```

Comment:
Here are my science results:
```
`'`: 7 backslashes in drawtext text param, 3 in fontfile; 3 in param names; 1 in filter name
`\`: 7 backslashes in drawtext text param, 3 in fontfile; 3 in param names; 1 in filter name
`:`: 2 backslashes
`,`: 1 backslash
`[`: 1 backslash
`]`: 1 backslash
`=`: 1 backslash
```
I was thinking something like this:
```
args = [escape_chars(str(x, '=:')) for x in self._args]
kwargs = {}
for k, v in self._kwargs.items():
    k = escape_chars(k, '\\\'=:')
    v = escape_chars(str(v), '\\\'=:')
    kwargs[k] = v

arg_params = ['{}'.format(arg) for arg in args]
kwarg_params = ['{}={}'.format(k, kwargs[k]) for k in sorted(kwargs)]
params = ':'.join(arg_params + kwarg_params)

params_text = self._name
if params:
    params_text += '={}'.format(':'.join(params))
return escape_chars(params_text, '\\\'[],;')
```
And then another `escape_chars` for rendering the `text` param of `drawtext`. I think this produces similar results to what's above. But need more science